gap/PolygonalComplexes/properties.gi

```diff
@@ -57,7 +57,36 @@ InstallOtherMethod( IsClosedSurface, "for a twisted polygonal complex",
         return IsClosedSurface(complex); # Call the function above
     end
 );
-
+InstallMethod( IsMultiTetrahedralSphere, "for a simplicial surface and an edge",
+    [IsTwistedPolygonalComplex],
```

Comment:
In the line before you wrote that the method is just for simplicial surfaces that is not uniformly.